src/lib/expansion/index.ts

```diff
@@ -0,0 +1,41 @@
+import {NgModule, ModuleWithProviders} from '@angular/core';
+import {CommonModule} from '@angular/common';
+import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
```

Comment:
Is this **BrowserAnimationsModule** import a problem?
I merged this PR into master and did a build, but my App now has this error:

> Error: BrowserModule has already been loaded.

I modified `publish-build-artifacts.sh` to deploy to my repo and ran it.
See https://github.com/angular/material2-builds/compare/master...arlowhite:master
Load diff for **material.js**
`+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';`
You can see that this import has been added.

_ng version_

> @angular/cli: 1.0.4
> node: 6.9.1
> os: linux x64
> @angular/common: 4.1.3
> @angular/material: 2.0.0-beta.5-25de2f2